### PR TITLE
Project-setUp: Changes

### PR DESCRIPTION
**Pull request by Paul-tes**
- Added Html structre
- added style
- added button hover states
- add border shadow to focus score section
- added dynamic list using dom.

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @Paul-tes , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an updated README file.
- Good job using the correct template but please update with your project name and everything with information for the current project. 
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/styles/style.css [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+.main {
+  display: flex;
+  flex-direction: row;
+  padding: 2rem;
+  gap: 1rem;
+  background-color: #ffcd74d1;
+}
+
+.score-header {
+  display: flex;
```
Comment (koachcode) [inline src/styles/style.css:25]: - [optional] Good job implementing the requirements, however, you added styles _that_ are not required, as the requirements needed just a wireframe, Learning to follow requirements strictly is a good skill to have.

**Review by koachcode — APPROVED** [review]
Hi @Paul-tes , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/style.css (hunk 1 above)
Comment (Paul-tes) [reply src/styles/style.css:25]: okay thanks

**Pull request merged**